primitives/darwinia-core/src/lib.rs

```diff
@@ -261,11 +260,11 @@ impl Chain for DarwiniaLike {
 	}
 }
 
-/// Convert a 256-bit hash into an AccountId.
-pub struct AccountIdConverter;
-impl Convert<H256, AccountId> for AccountIdConverter {
-	fn convert(hash: H256) -> AccountId {
-		let evm_address: H160 = hash.into();
-		evm_address.into()
-	}
-}
+// Convert a 256-bit hash into an AccountId.
```

Comment:
I don't know where this is used. I feel it is not need anymore.

Reply:
https://github.com/darwinia-network/darwinia-messages-substrate/blob/polkadot-v0.9.30/modules/dispatch/src/lib.rs#L268

Reply:
my commented code is different from the H256 -> H160.
I will write a test for this.

Reply:
do you mean this one: 
https://github.com/darwinia-network/darwinia-messages-substrate/blob/4655b8540d599294b6711089b4e6440ce4964d6c/modules/dispatch/src/lib.rs#L533

Reply:
This struct is used in the runtime of darwinia-2.0, not this crate. https://github.com/darwinia-network/darwinia-2.0/blob/main/runtime/crab/src/pallets/bridge_messages.rs#L44. Let me fix it.